utils/DfUtils.py

```diff
@@ -3,6 +3,9 @@
 '''
 import pandas as pd
 import uproot
+import ROOT
+import numpy as np
+from alive_progress import alive_bar
```

Comment:
It has to be added in the [requirements.txt](https://github.com/DmesonAnalysers/DmesonAnalysis/blob/master/requirements.txt) so that it's easier to install all the requirements in one shot, and possibly also to the [README.md](https://github.com/DmesonAnalysers/DmesonAnalysis/blob/master/README.md)